chain-impl-mockchain/src/evm/mod.rs

```diff
@@ -115,56 +101,38 @@ impl Decodable for EvmTransaction {
 #[cfg(feature = "evm")]
 impl Encodable for EvmTransaction {
     fn rlp_append(&self, s: &mut RlpStream) {
-        use EvmTransaction::*;
-        match self {
-            Create {
-                caller,
-                value,
-                init_code,
-                gas_limit,
-                access_list,
-            } => {
+        match &self.action_type {
+            EvmActionType::Create { init_code } => {
                 s.begin_list(6);
```

Comment:
I think there are 7 values below, but I would expect this to show up in tests. Maybe it's not covered?

Reply:
As I see, generally it will not a mistake to put less value than actually needed, It something similar as `Vec::with_capacity();`.